ADALiOS/ADALiOS/ADPolicyItem.h

```diff
@@ -0,0 +1,18 @@
+//
+//  ADPolicyItem.h
+//  ADALiOS
+//
+//  Created by Brandon Werner on 9/9/15.
+//  Copyright © 2015 MS Open Tech. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface ADPolicyItem : NSObject
+
+@property (strong) NSString* policyName;
+@property (strong) NSString* policyID;
```

Comment:
Once again this is a prototype thing that actually ended up not being used, but existed perviously when I had to convert the params that could be passed in to a Dictionary that can be passed as query params as NSURL. This is no longer needed, so it is gone from here and `ADAuthenticationRequest+WebRequest.m`.